packit_service/service/api/copr_builds.py

```diff
@@ -60,7 +60,8 @@ def get(self):
                     "project": build.project_name,
                     "owner": build.owner,
                     "build_id": build.build_id,
-                    "status": build.status,
+                    "status": build.status,  # Legacy, remove later.
+                    "status_per_chroot": {},
```

Comment:
I wouldn't personally mind breaking the API here and just set the dict to status

others may have a different opinion

Reply:
I'll just build tests for the entire API. It was long overdue anyways #489

Reply:
> I wouldn't personally mind breaking the API here and just set the dict to status
> 
> others may have a different opinion

So should I remove status and chroots, both are redundant if status_per_chroot exists.

Reply:
since no one from the team raised an opinion here, let's keep it as it is now